setup.py

```diff
@@ -20,6 +19,7 @@ def run(self, *args, **kwargs):
     scripts=[
         'calculator_app.py',
         'service.py',
-        'README.md'],
-    install_requires=['Flask']
-)
+        'README.md',
+        'commit.sh', 'newcommit.txt'],
```

Comment:
you don't need `commit.sh` in your package

Reply:
Fixed